### DISCUSSION TITLE
[BUG] No way to select code in NPM docs

### DISCUSSION THREAD

**Discussion by OnkelTem** [npm/feedback#876] [Website] [answered]
I cannot select any code in the docs.

For example, [here](https://docs.npmjs.com/cli/v8/commands/npm-version#description):

![image](https://user-images.githubusercontent.com/114060/221628348-dc638c7a-76c5-4130-8bf0-b2a83fbc8f20.png)

I cannot use the mouse to select anything from this snippet.

Firefox 110.0 (64-bit)

**Comment by monishcm** [comment, marked as answer]
I understand the text cannot be selected. Does the copy button at the top right work for you?

![image](https://user-images.githubusercontent.com/23190045/226343464-32078a24-d1a4-4f13-b61b-aadd2404b073.png)
